CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## v0.56.0
 
+FEATURES:
+* `r/tfe_team`: Add attribute `manage_agent_pools` to `organization_access` on `tfe_team` by @emlanctot [#1358](https://github.com/hashicorp/terraform-provider-tfe/pull/1358)
+
```

Comment:
I _believe_ this needs to be moved in the [unreleased](https://github.com/hashicorp/terraform-provider-tfe/blob/main/CHANGELOG.md#unreleased) section, as `v0.56.0` looks like the latest release!